applicationinsights.ts

```diff
@@ -69,15 +69,15 @@ let _performanceLiveMetrics: AutoCollectPerformance;
  * Initializes the default client. Should be called after setting
  * configuration options.
  *
- * @param instrumentationKey the instrumentation key to use. Optional, if
+ * @param setupString the Connection String or Instrumentation Key to use. Optional, if
  * this is not specified, the value will be read from the environment
- * variable APPINSIGHTS_INSTRUMENTATIONKEY.
+ * variable APPINSIGHTS_CONNECTIONSTRING or APPINSIGHTS_INSTRUMENTATIONKEY.
```

Comment:
This is the wrong environment variable. Please use "APPLICATIONINSIGHTS_CONNECTION_STRING"

Reply:
Resolved in 91297e5